polaris/ocean/convergence/forward.py

```diff
@@ -0,0 +1,145 @@
+from polaris.ocean.model import OceanModelStep, get_time_interval_string
+
+
+class ConvergenceForward(OceanModelStep):
+    """
+    A step for performing forward ocean component runs as part of a spherical
+    convergence test
+
+    Attributes
+    ----------
+    resolution : float
+        The resolution of the (uniform) mesh in km
+
+    package : Package
+        The package name or module object that contains the YAML file
+
+    yaml_filename : str
+        A YAML file that is a Jinja2 template with model config options
+
+    """
+
+    def __init__(self, component, name, subdir, resolution, mesh, init,
+                 package, yaml_filename='forward.yaml', options=None,
+                 graph_filename=`graph.info`, output_filename='output.nc',
```

Comment:
```suggestion
                 graph_filename='graph.info', output_filename='output.nc',
```
Sorry, wrong quotes.